src/org/labkey/test/tests/FileBasedPipelineTest.java

```diff
@@ -72,6 +73,28 @@ public void startTest()
         goToProjectHome();
     }
 
+    @Test
+    public void testScriptBasedFilewatchers()
+    {
+        String pipelineTaskWithHelpText = "Pipeline task with help text";
+        String helpText = "Hello! I am help text.";
+        String pipelineTaskNoHelpText = "two-steps";
+        String pipelineTaskInvisible = "Task that is not evokable by filewatcher";
+
+        goToFolderManagement().goToImportTab();
+        clickAndWait(Locator.lkButton("Manage file watcher triggers"));
+        DataRegionTable pipelineTable = new DataRegionTable("query", getDriver());
+        pipelineTable.clickInsertNewRow();
+
+        selectOptionByText(Locator.id("pipelineTaskSelect"), pipelineTaskWithHelpText);
+        Locator.byClass("lk-trigger-help-text").containing(helpText);
+
+        Locator.id("pipelineTaskSelect").notContaining(pipelineTaskInvisible);
```

Comment:
`Locator`s are just glorified `StringBuilder`s. They don't actually do anything until you attempt to use them to find something on the page. (e.g. `assertElementPresent` or `findElement`)

Reply:
Gotcha, updated!